Tasks/DockerV1/containerbuild.ts

```diff
@@ -66,5 +66,11 @@ export function run(connection: ContainerConnection): any {
         context = tl.getPathInput("buildContext");
     }
     command.arg(context);
+
+    command.on("stdout", data => {
```

Comment:
This should happen at the end (`stdout` event can happen in chunks).
On `stdout` event, append to a local variable and write to file and set the variable after command execution completes.